internal/controller/rbac/provider/roles/reconciler.go

```diff
@@ -296,27 +296,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 				continue
 			}
 
-			inFamily[member.GetUID()] = true
-		}
-	}
-
-	l := &extv1.CustomResourceDefinitionList{}
-	if err := r.client.List(ctx, l); err != nil {
```

Comment:
I thought removing this might mean the RBAC manager would not itself need RBAC access to get and list CRDs anymore. It turns out it does need them for its init container.